pkg/validations/cluster_node.go

```diff
@@ -0,0 +1,89 @@
+package validations
+
+import (
+	"fmt"
+)
+
+const (
+	singleAZCount = 1
+	MultiAZCount  = 3
```

Comment:
Why is one const exported and the other is not? 

Since it is being shared maybe both should be exported, so they can be used outside of this package if needed.

Reply:
Yeah, that is what i was thinking, since it is moved to this lib, it would be good to export them, such that we can use them if needed

Reply:
Changed singleAZCount to SingleAZCount